fastkafka_gen/cli.py

```diff
@@ -93,6 +122,16 @@ def generate_fastkafka_app(
 
 \n"""
     ),
+    input_path: str = typer.Option(
+        None,
+        "--input_file",
+        "-i",
+        help="""
+        The path to the file with the app desription. This path should be relative to the current working directory.
+        
+        \n\nWhen the user provides the application description both by via the file and through the CLI argument, the CLI argument takes precedence.
```

Comment:
Please re-word this second sentence. Something like below:

"If the app description is passed via both a file and a command line argument, the description from the command line will be used to create the application."